main.go

```diff
@@ -76,6 +77,9 @@ func main() {
 	root.Flags().StringVar(&upgradeConfig.MachineUpdates.Image.Field, "image-field",
 		"", "The image identifier field in provider manifests (optional)")
 
+	root.Flags().StringVar(&upgradeConfig.UpgradeID, "upgrade-id", fmt.Sprintf("%d", time.Now().Unix()),
```

Comment:
I'm wondering if having the defaulting logic for `upgradeConfig.UpgradeID` here is the right place for it? This is the front-end CLI and it might not always be used. How would you feel about putting the defaulting into `newBase` (for now) if the field is unset by the time we get there?

Reply:
Yep, that makes sense, putting logic in the default value is not great